### DISCUSSION TITLE
logout npm script should be renamed

### DISCUSSION THREAD

**Discussion by ctrlc03** [quadratic-funding/mpc-phase2-suite#265] [General]
Running `yarn logout` will trigger an actual logout from `yarn` not from the cli app. `npm run logout` works as expected. 
Renaming this will prevent any misunderstandings.

**Comment by 0xjei** [comment]
This problem happens only when using npm scripts, so do not affect the `phase2cli logout` command itself. BTW, I'm open to use different terms for this command, maybe we could usethe opposite of `auth` command (e.g., `deauth` or something similar).

**Reply by ctrlc03** [reply]
Yes sure we would only need changing the npm script command (changed the name of the discussion/issue). `deauth` could work, or what about a prefix  like `user` to each user command? e.g. `user:auth` - `user:logout` etc. These would include commands that both coordinator and user can run as opposed to `coordinate:commad`.

**Reply by 0xjei** [reply]
Tbh I do not like too much a prefix for general commands (it would be an extra layer not too 'justified', I would rather have a simple command execution, like `phase2cli auth`). I think a renaming is more than fine. Maybe we are going to set up a brainstorming session on naming and storytelling in the next few days.

**Reply by ctrlc03** [reply]
sounds good
